docs/reference/api/connectivity/nfc/NFCController.md

```diff
@@ -0,0 +1,49 @@
+## NFC controller
+
+To use an NFC controller, you must initiate the instance with a driver instance, an event queue and a scratch buffer for NDEF messages.
```

Comment:
This tells us how to use it but doesn't say what it is. Could we add a sentence saying what this is and when/why you'd want to use it?